### ISSUE TITLE
Implement Admin Accounts API (Part 2/2)

### ISSUE THREAD

**Issue by andregasser** [andregasser/bigbone#324]
Mastodon's Admin Accounts API allows an admin to perform moderation action with accounts. The goal of this task is to implement the following endpoints of this API in BigBone:

- Delete an account
- Perform an action against an account
- Enable a currently disabled account
- Unsilence an account
- Unsuspend an account
- Unmark an account as sensitive

The Admin Accounts API as it is documented by Mastodon can be found here:
https://docs.joinmastodon.org/methods/admin/accounts/

Tip: A lot of other endpoints similar to this have already been implemented. In order to get an idea how to implement this, I suggest that you study the code upfront. Start with one of the `*Methods.kt` files.

Please consult the following documents to make sure your contribution meets our projects criteria:

- [Code of Conduct](https://github.com/andregasser/bigbone/blob/master/CODE_OF_CONDUCT.md)
- [Code Contribution Guidelines](https://github.com/andregasser/bigbone/blob/master/CONTRIBUTING.md)

**Comment by andregasser** [comment]
Whoever is implementing this, should not start with this, before https://github.com/andregasser/bigbone/issues/323 is merged into master, as some base entities will be created in that story that are required here as well (to avoid duplicate work).
